contributors.yaml

```diff
@@ -71,7 +71,7 @@ contributors:
     given-names: Alyssa
     affiliation: "University of Michigan"
     orcid: "https://orcid.org/0000-0001-5953-8807"
-
+  -
```

Comment:
Was there a syntax error here?

Reply:
I think it may be valid YAML syntax but was not valid for the schema needed by `.sync-zenodo-metadata.py`. It failed when I ran that script:

```python
Traceback (most recent call last):
  File "./.sync-zenodo-metadata.py", line 87, in <module>
    sync()
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "./.sync-zenodo-metadata.py", line 60, in sync
    contributors = load(file.read(), Loader=Loader)['contributors']
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/ruamel/yaml/main.py", line 949, in load
    return loader._constructor.get_single_data()
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/ruamel/yaml/constructor.py", line 113, in get_single_data
    return self.construct_document(node)
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/ruamel/yaml/constructor.py", line 123, in construct_document
    for _dummy in generator:
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/ruamel/yaml/constructor.py", line 723, in construct_yaml_map
    value = self.construct_mapping(node)
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/ruamel/yaml/constructor.py", line 440, in construct_mapping
    return BaseConstructor.construct_mapping(self, node, deep=deep)
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/ruamel/yaml/constructor.py", line 257, in construct_mapping
    if self.check_mapping_key(node, key_node, mapping, key, value):
  File "/home/bdice/miniconda3/envs/dice/lib/python3.7/site-packages/ruamel/yaml/constructor.py", line 295, in check_mapping_key
    raise DuplicateKeyError(*args)
ruamel.yaml.constructor.DuplicateKeyError: while constructing a mapping
  in "<byte string>", line 70, column 5:
        family-names: Travitz
        ^ (line: 70)
found duplicate key "family-names" with value "Butler" (original value: "Travitz")
  in "<byte string>", line 75, column 5:
        family-names: Butler
        ^ (line: 75)

To suppress this check see:
    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys

Duplicate keys will become an error in future releases, and are errors
by default when using the new API.
```

Reply:
Ok, but with that change it works fine?

Reply:
Yes, seems fine after fixing that.